test/sdil/utils/FakeApplicationSpec.scala

```diff
@@ -16,20 +16,183 @@
+    //when(mockSdilConnector.returns).thenReturn(returnsMock)
+    val aSubscription =  RetrievedSubscription(
+      "0000000022",
+      "XKSDIL000000022",
+      "Super Lemonade Plc",
+      UkAddress(List("63 Clifton Roundabout", "Worcester"),"WR53 7CX"),RetrievedActivity(false,true,false,false,false),LocalDate.of(2018,4,19),List(Site(UkAddress(List("33 Rhes Priordy", "East London"),"E73 2RP"),Some("88"),Some("Wild Lemonade Group"),Some(LocalDate.of(2018,2,26))), Site(UkAddress(List("117 Jerusalem Court", "St Albans"),"AL10 3UJ"),Some("87"),Some("Highly Addictive Drinks Plc"),Some(LocalDate.of(2019,8,19))), Site(UkAddress(List("87B North Liddle Street", "Guildford"),"GU34 7CM"),Some("94"),Some("Monster Bottle Ltd"),Some(LocalDate.of(2017,9,23))), Site(UkAddress(List("122 Dinsdale Crescent", "Romford"),"RM95 8FQ"),Some("27"),Some("Super Lemonade Group"),Some(LocalDate.of(2017,4,23))), Site(UkAddress(List("105B Godfrey Marchant Grove", "Guildford"),"GU14 8NL"),Some("96"),Some("Star Products Ltd"),Some(LocalDate.of(2017,2,11)))),List(),Contact(Some("Ava Adams"),Some("Chief Infrastructure Agent"),"04495 206189","Adeline.Greene@gmail.com"),None)
+
+    val sdilReturn = SdilReturn((0,0), (0,0), List.empty, (0,0), (0,0), (0,0), (0,0))
+
+    lazy val cacheMock = mock[ShortLivedHttpCaching]
+
+    lazy val mockSdilConnector: SoftDrinksIndustryLevyConnector = {
+      val m = mock[SoftDrinksIndustryLevyConnector]
+      when(m.submit(any(),any())(any())).thenReturn(Future.successful(()))
+      when(m.retrieveSubscription(any(),any())(any())).thenReturn(Future.successful(None))
+      when(m.retrieveSubscription(matching("XZSDIL000100107"),any())(any())).thenReturn(Future.successful(Some(aSubscription)))
+      when(m.returns_pending(any())(any())).thenReturn(Future.successful(Nil))
+      when(m.returns_variable(any())(any())).thenReturn(Future.successful(returnPeriods))
+      when(m.returns_vary(any(), any())(any())).thenReturn(Future.successful(()))
+      when(m.returns_update(any(), any(), any())(any())).thenReturn(Future.successful(()))
+      when(m.returns_get(any(),any())(any())).thenReturn(Future.successful(None))
+      when(m.returns_variation(any(),any())(any())).thenReturn(Future.successful(()))
+      when(m.submitVariation(any(),any())(any())).thenReturn(Future.successful(()))
+      when(m.balanceHistory(any(),any())(any())).thenReturn(Future.successful(Nil))
+      when(m.balance(any(),any())(any())).thenReturn(Future.successful(BigDecimal(0)))
+      when(m.shortLiveCache) thenReturn cacheMock
+      when(cacheMock.fetchAndGetEntry[Any](any(),any())(any(),any(),any())).thenReturn(Future.successful(None))
+      when(m.checkSmallProducerStatus(any(), any())(any())) thenReturn Future.successful(None)
+      when(m.submit(any(), any())(any())) thenReturn Future.successful(())
+      m
+    }
+
+    lazy val mockRegistrationFormDataCache = {
+
+    }
+
+    //  lazy val mockSubscription: Subscription.type = {
```

Comment:
REmove the comments after we merge and we are sure the acceptance tests pass

Reply:
these comments that you see were existing ones, so decided to leave them. Do you think they should still be removed?